src/main/java/net/simpvp/Misc/DisableCmd.java

```diff
@@ -88,13 +88,28 @@ public void onServerCommandEvent(ServerCommandEvent event) {
 		String cmd = split[0].trim().toLowerCase();
 		Cmd c = cmds.get(cmd);
 
+		boolean allowCommand = true;
+		if (cmd.equals("kill") || cmd.equals("tp")) {
+			for (String args: split) {
+				if (args.startsWith("@") && !(args.contains("distance=..") || args.contains("distance=0.."))) {
```

Comment:
There are other parameters such as `dx` and `x`. I think they should work as well?